db/config.cc

```diff
@@ -799,26 +799,8 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     */
     , broadcast_rpc_address(this, "broadcast_rpc_address", value_status::Used, {/* unset */},
         "RPC address to broadcast to drivers and other Scylla nodes. This cannot be set to 0.0.0.0. If blank, it is set to the value of the rpc_address or rpc_interface. If rpc_address or rpc_interfaceis set to 0.0.0.0, this property must be set.\n")
-    , rpc_port(this, "rpc_port", "thrift_port", value_status::Used, 9160,
-        "Thrift port for client connections.")
-    , start_rpc(this, "start_rpc", value_status::Used, false,
-        "Starts the Thrift RPC server")
     , rpc_keepalive(this, "rpc_keepalive", value_status::Used, true,
         "Enable or disable keepalive on client connections (RPC or native).")
```

Comment:
The "RPC" mentioned here, is it really Scylla's RPC, or the Thrift that we're now deleting and the word should be deleted from the description?